include/SFML/System/Clock.hpp

```diff
@@ -74,11 +76,26 @@ class SFML_SYSTEM_API Clock
     Time restart();
 
 private:
+    using ClockImpl = std::conditional_t<std::chrono::high_resolution_clock::is_steady,
+                                         std::chrono::high_resolution_clock,
+                                         std::chrono::steady_clock>;
+    static_assert(ClockImpl::is_steady, "Provided implementation is not a monotonic clock");
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Convert clock duration to microseconds
+    ///
+    /// This function acts as a utility for converting clock
+    /// duration type instance into sf::Time
+    ///
+    /// \return Time in microseconds
+    ///
+    ////////////////////////////////////////////////////////////
+    static Time durationToTime(ClockImpl::duration duration);
```

Comment:
```suggestion
    [[nodiscard]] static Time durationToTime(ClockImpl::duration duration);
```

Reply:
@reconndev: this was resolved, but I don't see the change applied.

Reply:
^

Reply:
Must have been reverted accidentally. Done